libraries/Microsoft.Bot.Builder/PathUtils.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Microsoft.Bot.Builder
+{
+    public static class PathUtils
```

Comment:
I'd prefer PathResolver or some verb rather than utils...when code grow people tend to treat *Utils classes as places where they can drop any random method